misc/prime.c

```diff
@@ -1,35 +1,67 @@
+/**
+ * @file
+ * @brief Program to identify if a number is [prime
+ * number](https://en.wikipedia.org/wiki/Prime_number) or not
+ */
+#include <assert.h>
 #include <math.h>
+#include <stdbool.h>
 #include <stdio.h>
 
-int isPrime(int x)
+/**
+ * Check if a given number is prime number or not
+ * @param x number to check
+ * @return `true` if given number is prime number, otherwise `false`
+ */
+bool isPrime(int x)
 {
     if (x == 2)
     {
-        return 1;
+        return true;
     }
     if (x < 2 || x % 2 == 0)
     {
-        return 0;
+        return false;
     }
 
     double squareRoot = sqrt(x);
 
     for (int i = 3; i <= squareRoot; i += 2)
     {
         if (x % i == 0)
-            return 0;
+        {
+            return false;
+        }
     }
-    return 1;
+    return true;
 }
 
+/**
+ * Test function
+ */
```

Comment:
```suggestion
 * @return void
 */
```
It seems that doxygen is unable to document the functions if the docstring does not contain a `@return` or `@param` identifiers. Can you add them here and for `main`.

Reply:
Thanks 👍